Source/Controllers/Cells/DetailTableViewCell.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  DetailTableViewCell.swift
+//  VirgilMessenger
+//
+//  Created by Matheus Cardoso on 2/21/20.
+//  Copyright © 2020 VirgilSecurity. All rights reserved.
+//
+
+import UIKit
+
+class DetailTableViewCell: UITableViewCell {
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: .value1, reuseIdentifier: reuseIdentifier)
```

Comment:
Any time I call this initializer, style parameter will be ignored. Bad practice.

Reply:
Why do we need this class at all. All it does is initializing UITableViewCell with different parameter

Reply:
To make it possible to register as reusable cell: https://github.com/VirgilSecurity/virgil-messenger-x/pull/3/files#diff-3d3e5f4139b3583941446a9f37e2e3f5R65

Do you know a better way?

Reply:
Not that we need reusable cells in this simple table view... so I could remove cell reusability

Reply:
Ok